### PR TITLE
feat: KstreamsRebalanceListener added as default rebalance listener

### PR DESCRIPTION
**Pull request by marcosschroh**

### CODE REVIEW

File: docs/stream.md [hunk 1]
```diff
@@ -237,6 +237,20 @@ For some cases you will need a `RebalanceListener` so when partitions are `assig
 - Saving offsets in a custom store when a partition is `revoked`
 - Load a state or cache warmup on completion of a successful partition re-assignment.
 
+If you do not provide a `RebalanceListener` kstreams will set the default `KstreamsRebalanceListener`. If `manual` commit is enabled, `KstreamsRebalanceListener` will call `commit`
```
Comment (woile) [inline docs/stream.md:240]: Somewhere here add a link to the manual commit example.

**Review by woile — APPROVED** [review]
I would also like to see a "batch commit" example

**Comment by marcosschroh** [comment]
> I would also like to see a "batch commit" example

Do you mean something like this:

```python
batch = []

async def process_msg_batch(a_batch: List[...]):
    ...

@stream_engine.stream(
    topics=["local--hello-world"],
    group_id="example-group",
    enable_auto_commit=False,  # it means that we need to call commit
)
async def consume(stream):
    async for cr in stream:
        batch.append(msg)
        if len(batch) == 100:
            await process_msg_batch(batch)
            await consumer.commit()
            batch = []

        await stream.commit()
```

**Comment by woile** [comment]
Yes, something like that

**Pull request merged**